### DISCUSSION TITLE
Tx power and report frequency.

### DISCUSSION THREAD

**Discussion by artur1702** [pvvx/THB2#27] [Q&A]
Hi,
I can't find it in the documentation. Setting Tx Power +5 dBm increases the range?
Which parameter is responsible for the frequency of temperature and humidity reporting? A report every few minutes would be enough for me.

Thanks

**Comment by pvvx** [comment]
All settings are made in  [PHY62x2BTHome.html](https://pvvx.github.io/THB2/web/PHY62x2BTHome.html).

**Reply by AleXSR700** [reply]
I think the question was more towards the advertising frequency.

I was wondering something similar.
Is there a minimum advertising frequency to be able to connect?
That's the case for the Xiaomi devices and I am looking for best compromise between reachability and battery life.

**Reply by pvvx** [reply]
Bluetooth SIG standard - interval up to 10 seconds. This is normal for many Android smartphones.
Main problems in Linux. Linux has been intentionally introducing bugs for many years to make it impossible to use Bluetooth.

Early implementations of the Linux kernel had 10 seconds. Then 4 seconds, it came to 2. Now you have to install it yourself by tinkering with the Bluez settings. The default Bluetooth settings in Linux do not comply with the Bluetooth SIG standard in many ways. After 5 years, they have already begun to partially correct it. Wait another 5 years.
